test/config.spec.js

```diff
@@ -16,11 +16,18 @@ describe('Config', function() {
 
   var CUSTOM_INVALID =
     '{' +
-    '  "colors": {' +
-    '    "text": "pretty" ' +
+    '"themes": {' +
+    '  "simple": {' +
+    '    "name": "bold,underline",' +
+    '    "description": "#876992",' +
```

Comment:
would it make sense to change the variables "name", "description" to "commandName", "mainDescription" on this PR, or would that require touching much more of the program?